lib/tasks/publishing_api.rake

```diff
@@ -27,7 +27,7 @@ namespace :publishing_api do
       {
         content_id: 'a6d9bafd-f69b-4d2f-a002-c7547473e152',
         current_base_path: '/feedback/foi',
-        destination_path: '/contact/foi'
+        destination_path: '/make-a-freedom-of-information-request'
       },
       {
```

Comment:
Do I also need to do:
```
{
        content_id: 'a6d9bafd-f69b-4d2f-a002-c7547473e152',	        
        current_base_path: '/contact/foi',
        destination_path: '/make-a-freedom-of-information-request'
},
```

Reply:
You'll need two redirections.  One from `/feedback/foi` to `/contact/foi` and another from `/contact/foi` to `/make-a-freedom-of-information-request`.